--- a/chrome/browser/media/webrtc/capture_policy_utils.cc
+++ b/chrome/browser/media/webrtc/capture_policy_utils.cc
@@ -181,27 +181,11 @@ void RegisterProfilePrefs(PrefRegistrySi
 
 void CheckGetAllScreensMediaAllowedForAnyOrigin(
     base::OnceCallback<void(bool)> callback) {
-#if BUILDFLAG(IS_CHROMEOS)
-  if (crosapi::mojom::MultiCaptureService* multi_capture_service =
-          GetMultiCaptureService()) {
-    multi_capture_service->IsMultiCaptureAllowedForAnyOriginOnMainProfile(
-        std::move(callback));
-    return;
-  }
-#endif  // BUILDFLAG(IS_CHROMEOS)
   std::move(callback).Run(/*result=*/false);
 }
 
 void CheckGetAllScreensMediaAllowed(const GURL& url,
                                     base::OnceCallback<void(bool)> callback) {
-#if BUILDFLAG(IS_CHROMEOS)
-  crosapi::mojom::MultiCaptureService* multi_capture_service =
-      GetMultiCaptureService();
-  if (multi_capture_service) {
-    multi_capture_service->IsMultiCaptureAllowed(url, std::move(callback));
-    return;
-  }
-#endif  // BUILDFLAG(IS_CHROMEOS)
   std::move(callback).Run(/*result=*/false);
 }
 
